lnwallet/chanfunding/coin_select.go

```diff
@@ -156,10 +156,13 @@ func sanityCheckFee(totalOut, fee btcutil.Amount, maxFeeRatio float64) error {
 			"got %.2f", maxFeeRatio)
 	}
 
+	maxFee := btcutil.Amount(int64(float64(totalOut) * maxFeeRatio))
+
 	// Check that the fees do not exceed the max allowed value.
-	if float64(fee) > float64(totalOut)*maxFeeRatio {
-		return fmt.Errorf("fee %v on total output value %v with max "+
-			"fee ratio of %.2f", fee, totalOut, maxFeeRatio)
+	if fee > maxFee {
+		return fmt.Errorf("fee %v exceeds max fee (%v) on total "+
```

Comment:
could we maybe add the required ratio here as well: fee/totalOut?

Reply:
was just thinking of making it easier for the user to find out the right ratio he needs to put in (fee/totalOut) in case it fails, otherwise maybe the user can do it by himself according to the output

Reply:
oh I see what you mean now

My only issue is that the ratio is meant to dictate what the fees/outputs are going to be like, not the other way around

the scenario of "oh the psbt fund failed, what ratio do i need?" kind of defeats the purpose of the (protective) ratio in the first place

for the cmd superuser case we don't really need to give them the exact min ratio required for this to succeed (ie fee/totalOut) as they can just use `1.00`, since they know what they're doing